definitions/npm/moment_v2.x.x/flow_v0.25.x-/moment_v2.x.x.js

```diff
@@ -118,30 +118,51 @@ declare class moment$MomentDuration {
 }
 declare class moment$Moment {
   static ISO_8601: string;
+  static (): moment$Moment;
   static (
-    initDate: ?Object | number | Date | Array<number> | moment$Moment | string
+    initDate: moment$MomentOptions | number | Date | Array<number> | moment$Moment | string
```

Comment:
It's funny that the `moment$MomentOptions` was already defined in this file, but it just wasn't ever used before.

https://github.com/flow-typed/flow-typed/blob/08b5253e98d12eeada6066cabcf003b4e73effc9/definitions/npm/moment_v2.x.x/flow_v0.25.x-/moment_v2.x.x.js#L1-L24